firmware/common2015/utils/logger/logger.hpp

```diff
@@ -24,6 +25,8 @@
 #define LOG(lvl, ...) \
     log(lvl, __BASE_FILE_NAME__, __LINE__, __func__, __VA_ARGS__)
 
+#define S_LOG(lvl) LogHelper(lvl, __BASE_FILE_NAME__, __LINE__, __func__)
```

Comment:
Actually, in order for it to show up on our docs site, use three slashes `///` for the comment or use a multiline comment starting with `/**`.  Doxygen ignores comments with only two slashes.